.github/workflows/build.yml

```diff
@@ -83,9 +89,7 @@ jobs:
     - name: install monero dependencies
       run: ${{env.APT_INSTALL_LINUX}}
     - name: build
-      run: |
-        ccache --max-size=150M
-        make -j3
+      run: ${{env.BUILD_DEFAULT_LINUX}}
```

Comment:
Why? this just makes it more complicated for no apparent reason.

Reply:
We need to be able to reuse the exact cmake command, as explained in the other comment.